client/src/App.js

```diff
@@ -1,18 +1,25 @@
-import React, { useState } from 'react';
+import React, { useState } from "react";
+import { Route } from "react-router-dom";
 
-import SavedList from './Movies/SavedList';
+import SavedList from "./Movies/SavedList";
+import Movie from "./Movies/Movie";
+import MovieList from "./Movies/MovieList";
 
 const App = () => {
-  const [savedList, setSavedList] = useState( [] );
+  const [savedList, setSavedList] = useState([]);
 
   const addToSavedList = movie => {
-    setSavedList( [...savedList, movie] );
+    !savedList.includes(movie) && setSavedList([...savedList, movie]);
```

Comment:
I had another student on my 5th day who used .includes.  He had a weird effect where it wouldn't work when ie migrated to home back to MovieCard.  Meaning it would add it a second time.  I'll have to check it with yours.